src/components/Notification/ToastNotification/_index.scss

```diff
@@ -4,90 +4,4 @@
 /// @copyright IBM Security 2019
 ////
 
-@import '../../../globals/index';
-
-@include export-namespace($name: toast-notification) {
-  .#{$prefix}--toast-notification {
-    display: flex;
-    width: rem(288px);
-    height: auto;
-    padding-left: $carbon--spacing-05;
-    color: $carbon--gray-100;
-    margin-top: $carbon--spacing-03;
-    margin-bottom: $carbon--spacing-03;
-    margin-right: $carbon--spacing-05;
-    background: $carbon--white-0;
-
-    &:first-child {
-      margin-top: $carbon--spacing-05;
-    }
-  }
-
-  .#{$prefix}--toast-notification--error {
-    border-left: 3px solid $carbon--red-50;
-  }
-
-  .#{$prefix}--toast-notification--success {
-    border-left: 3px solid $carbon--green-50;
-  }
-
-  .#{$prefix}--toast-notification--info {
-    border-left: 3px solid $carbon--blue-50;
-  }
-
-  .#{$prefix}--toast-notification--warning {
-    border-left: 3px solid $carbon--yellow-20;
-  }
-
-  .#{$prefix}--toast-notification__details {
-    margin-right: $carbon--spacing-05;
-  }
-
-  .#{$prefix}--toast-notification__close-button {
-    background-color: transparent;
-    border: none;
-    cursor: pointer;
-    margin-left: auto;
-    padding: 0;
-    height: rem(48px);
-    width: rem(48px);
-    min-height: rem(48px);
-    min-width: rem(48px);
-
-    &:focus {
-      background-color: $carbon--gray-10;
-      outline: 0.1875rem solid $carbon--blue-60;
-      outline-offset: -0.1875rem;
-      border-width: 0;
-    }
-
-    &:hover {
-      background-color: $ui-05;
-    }
-
-    .#{$prefix}--toast-notification__close-icon {
-      height: 1rem;
-      width: 1rem;
-      fill: $carbon--gray-100;
-    }
-  }
-
-  .#{$prefix}--toast-notification__title {
-    @include carbon--type-style($name: body-short-01);
-    font-weight: 600;
-    margin-top: 1rem;
-  }
-
-  .#{$prefix}--toast-notification__subtitle {
-    @include carbon--type-style($name: body-short-01);
-    color: $carbon--gray-100;
-    margin-top: 0;
-    margin-bottom: $carbon--spacing-06;
-  }
-
-  .#{$prefix}--toast-notification__caption {
-    @include carbon--type-style($name: body-short-01);
-    color: $carbon--gray-100;
-    margin-bottom: $carbon--spacing-05;
-  }
-}
+@import 'carbon-components/scss/components/notification/toast-notification';
```

Comment:
Can we just delete this file now and import `ToastNotification` styles one level up, alongside the `InlineNotification` style import? 😅

Reply:
Good point! I'd like to keep the file just in case anyone is relying on it, but I can move the import up